packages/py3_snowballstemmer.rb

```diff
@@ -7,6 +7,7 @@ class Py3_snowballstemmer < Pip
   license 'BSD'
   compatibility 'all'
   source_url 'SKIP'
+  binary_compression 'tar.zst'
```

Comment:
Why is there a `binary_compression` value in this package if it does not have any `binary_sha256` values and it still has `no_compile_needed`?